src/config/configuration.ts

```diff
@@ -118,6 +136,23 @@ const configuration = () => {
       accessGroupProperty: process.env.OIDC_ACCESS_GROUPS_PROPERTY, // Example: groups
       autoLogout: process.env.OIDC_AUTO_LOGOUT || false,
       returnURL: process.env.OIDC_RETURN_URL,
+      userInfoMapping: {
+        id: process.env.OIDC_USERINFO_MAPPING_FIELD_ID,
+        username:
+          oidcUsernameFieldMapping.split(",").map((v) => v.trim()) ?? [],
+        displayName: process.env.OIDC_USERINFO_MAPPING_FIELD_DISPLAYNAME,
+        familyName: process.env.OIDC_USERINFO_MAPPING_FIELD_FAMILYNAME,
+        emails: process.env.OIDC_USERINFO_MAPPING_FIELD_EMAILS,
+        email: process.env.OIDC_USERINFO_MAPPING_FIELD_EMAIL,
+        thumbnailPhoto: process.env.OIDC_USERINFO_MAPPING_FIELD_THUMBNAIL_PHOTO,
+        groups: process.env.OIDC_USERINFO_MAPPING_FIELD_GROUPS,
```

Comment:
Do we need mapping for groups?
If so, does it duplicate the functionality that accessGroupProperty has?

Reply:
1. Yes, we need mapping for groups, because the property name of groups can be changed from IAM.
2. Yes, if we only consider ESS use case, it's duplicated.  `accessGroupProperty` is used for `access-group-from-payload`, where different users may have different preferences for property name of the groups. Especially in the scenario where a user have different types of groups for different purposes.

Reply:
1. Yes, we need mapping for groups because the property name of groups can be changed in IAM.
2. Yes, if we consider only the ESS use case, it is duplicated.`accessGroupProperty` is used for access-group-from-payload to set accessGroups from oidc response, wheras `OIDC_USERINFO_MAPPING_FIELD_GROUPS` is used in userProfile transformation function. In scenarios where a user has different types of groups for different purposes, it might be helpful to separate the value from each other .